usr/share/rear/conf/default.conf

```diff
@@ -622,6 +622,39 @@ BEXTRACT_VOLUME=
 BEXTRACT_DEVICE=
 
 ##
+# BACKUP=BORG stuff (https://borgbackup.readthedocs.io)
+##
+# Copy Borg stuff 1:1
+COPY_AS_IS_BORG=( )
+# Binaries Borg needs to operate correctly
+# ReaR includes `borg' and `locale' automatically
+PROGS_BORG=( )
+# Borg server name
+BORG_HOST=
+# Username for connection to Borg server
+BORG_USERNAME=
+# Path to Borg repository on Borg server
+BORG_REPO=
+# Prefix used by ReaR to name archives
+# WARNING:
+# Do not use '_' in BORG_ARCHIVE_PREFIX as it is used internally by ReaR.
+BORG_ARCHIVE_PREFIX="rear"
+# Compression used by Borg
+# Syntax: <compression_type,level>
+# compression_type: none, lz4, zlib, lzma
+# level: 0-9
+BORG_COMPRESSION="zlib,9"
```

Comment:
I assume with an empty
BORG_COMPRESSION=""
the Borg default gets used.

@gozora 
please add a comment to default.conf why
BORG_COMPRESSION=""
is the default to avoid enhancement requests
from other usres to use a "better" default here
(which could get easily implemented at some later time
in the future when nobody remembers this issue here).